BLL/Services/EmailService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using MailKit.Security;
+using Microsoft.Extensions.Configuration;
+using MimeKit;
+using MimeKit.Text;
+using System;
+using System.Threading.Tasks;
+using SmtpClient = MailKit.Net.Smtp.SmtpClient;
+
+namespace BLL.Services
+{
+    public class EmailService : IEmailService
+    {
+        private readonly IConfiguration _configuration;
+
+        public EmailService(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public async Task SendAdminEmail(Email email)
```

Comment:
+1 
dobrze,  że zwracasz Task (lub Task<...> )
Pamiętajcie żeby nie zwracać nigdy async VOID , bo jest problem z wyjątkami wtedy.
https://docs.microsoft.com/en-us/archive/msdn-magazine/2013/march/async-await-best-practices-in-asynchronous-programming